zipkin-cassandra-core/src/main/java/org/twitter/zipkin/storage/cassandra/Repository.java

```diff
@@ -205,6 +220,28 @@ public Repository(String keyspace, Cluster cluster, Boolean ensureSchema) {
                     .value("ts", QueryBuilder.bindMarker("ts"))
                     .value("trace_id", QueryBuilder.bindMarker("trace_id"))
                     .using(QueryBuilder.ttl(QueryBuilder.bindMarker("ttl_"))));
+
+        selectTraceIdsBySpanDuration = session.prepare(
+                QueryBuilder.select("d", "ts", "tid")
+                    .from("span_duration_index")
+                    .where(QueryBuilder.eq("s", QueryBuilder.bindMarker("service_name")))
```

Comment:
here's some anecdotal evidence (since it's hard to measure exactly)
> ... as an example, for data file sizes > 24h (and thus at the max level of compaction)
> 168829777409 with long column names
> 31931936664 with short column names